src/TBuffer.cpp

```diff
@@ -2754,12 +2754,12 @@ bool TBuffer::moveCursor(QPoint& where)
 // requested by lua function getLines(...):
 QString badLineError = qsl("ERROR: invalid line number");
 
-QString& TBuffer::line(int n)
+QString& TBuffer::line(int lineNumber)
 {
-    if ((n >= lineBuffer.size()) || (n < 0)) {
+    if ((lineNumber >= lineBuffer.size()) || (n < 0)) {
```

Comment:
On a micro-optimisation basis I'd put the easier to convert to assembly code test first:
```suggestion
    if ((lineNumber < 0) ||(lineNumber >= lineBuffer.size())) {
```